enarx-keep/tests/integration_tests.rs

```diff
@@ -17,7 +17,7 @@ const TIMEOUT_SECS: u64 = 5;
 
 /// Returns a handle to a child process through which output (stdout, stderr) can
 /// be accessed.
-fn run_test(bin: &str, status: i32) -> std::process::Child {
+fn run_test(bin: &str, status: i32, input: Option<&str>) -> std::process::Child {
```

Comment:
```suggestion
fn run_test(bin: &str, status: i32, input: impl Into<Option<&str>>) -> std::process::Child {
```

There's a neat Rust trick here. See below...